### DISCUSSION TITLE
Integration with time series database

### DISCUSSION THREAD

**Discussion by occoder** [online-ml/river#747] [Q&A]
As a streaming machine learning package, does River support integration with certain time series database, e.g. influxdb?
Such as supporting query from the time series database and take that query result as the input to River?

**Comment by MaxHalford** [comment]
Hello @occoder!

The answer is no, there is no integration with InfluxDB. It shouldn't be too difficult to do though. Indeed there's already a `stream.iter_sql` function for SQLAlchemy connections.

Do you feel up to contributing something? If not, I can try to look into it.
